rock-n-rollup/src/services/ticket_upgrade.rs

```diff
@@ -55,7 +57,7 @@ fn upgrade_on_ticket<R: Logger + Dac + Installer>(
 
 impl<R> IntoService<R, Vec<u8>, TicketUpgrade> for TicketUpgrade
 where
-    R: Runtime,
+    R: Runtime + 'static,
```

Comment:
is it needed because of the introduction of Runtime earlier in the code?

Reply:
I have remove the `static` at this file at this [commit](f62e693698d8b622c3351f686d7de6cd33b20721)